language/classify_text/classify_text_tutorial.py

```diff
@@ -0,0 +1,251 @@
+# Copyright 2017, Google, Inc.
```

Comment:
This needs a shebang: https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#shebang

Reply:
done.